modules/apps/portal-store/portal-store-gcs/src/test/resources/com/liferay/portal/store/gcs/dependencies/service-account-key.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Just to clarify any concerns: this key was generated online only for this example. It does not come from GCP and is not used for anything else.